dev_guide/projected_volumes.adoc

```diff
@@ -0,0 +1,368 @@
+[[dev-guide-projected-volumes]]
+= Projected Volumes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+A _projected volume_ maps several existing xref:volumes.adoc#dev-guide-volumes[volume sources] into the same directory.
+
+Currently, the following types of volume sources can be projected:
+
+* xref:secrets.adoc#dev-guide-secrets[Secrets]
+* xref:configmaps.adoc#dev-guide-configmaps[Config Maps]
+* xref:downward_api.adoc#dev-guide-downward-api[Downward API]
+
+[NOTE]
+====
+All sources are required to be in the same namespace as the pod.
+====
+
+Projected volumes allow the user to map any combination of these volume sources into a single directory
+
+* automatically populate a single volume with the keys from multiple secrets, configmaps, and with downward API information,
+so that I can synthesize a single directory with various sources of information
+* populate a single volume with the keys from multiple secrets, configmaps, and with downward API information,
+explicitly specifying paths for each item, so that I can have full control over the contents of that volume
```

Comment:
Should there be punctuation at the end of line 28? I'm not sure what to suggest exactly, because the sentence doesn't really run on to the two points above. Is there a way that can be clearer? I'd also suggest full stops after the points.